gnosis/safe-react
# Show execute transaction checkbox for unexecuted transaction

## What it solves
Resolves #2955

## How this PR fixes it
Show the execute transaction checkbox checkbox when the last transaction has yet to be executed.

## How to test it
Queue a transaction on a Safe which has >1 owners and has a threshold of >1. The threshold deciding signer should see the option to execute the transaction.
